db/schema.rb

```diff
@@ -685,6 +685,7 @@
     t.float "points", default: 1.0
     t.integer "reactable_id"
     t.string "reactable_type"
+    t.integer "reading_time", default: 0
```

Comment:
You shouldn't need to add `reading_time` to the reaction model. You can include it in the Algolia index without it having to be on the reaction have it in its own table—similar to the other models here like `reactable_published_date` etc.

Reply:
Removed it from the model and undid the migration.